qcodes_contrib_drivers/drivers/Bilt/ITest.py

```diff
@@ -101,22 +107,47 @@ def __init__(self, parent: Instrument,
     
         self.add_parameter('bilt_name',
                            set_cmd=partial(self._parent._set_chan_name, chan_num),
-                           set_parser=str)
+                           set_parser=str,
+                           initial_value=f'Chan{chan_num:02d}')
 
-        self.add_parameter('synchronous',
-                           get_cmd=partial(self._parent._get_chan_synchronous, chan_num),
+        self.add_parameter('synchronous_enable',
```

Comment:
If you want the instrument is initilized with this parameter, rather than explicitly putting it in the __init__ as a class argument and the creating only readable parameter later, you can create a settable parameter and get it under __init__ like:
def __init__(...):
  .
  .
  .
  self.synchronous_enable()
And don't forget to change synchronous_enable to self.synchronous_enable() anywhere else you used it.

Then if you want it to be initilized with True, you just simply set it to its initial value:
 
```
self.add_parameter('synchronous_enable',
                           docstring='Is the channel in synchronous mode.',
                           get_cmd=None,
                           get_parser=bool,
                           set_cmd=partial(self._parent._set_chan_synchronous, chan_num),
                           initial_value=True
                           vals=vals.Bool())
```

what you have done seems duplication of creating an argument and excatly the same parameter, which is set in init and get as a parameter.

Reply:
I wanted the user to have the possibility to change this parameter at the initialization of the device.
Is there another way of doing it?

Reply:
If you want to be forced at initialization, then there should not be any default value in it
Then you have it as an argument and pass it to the parameter under init, for example:

Def init(sync: bool):
  Self.enabled_synchronous(sync)


And you remove the initial value from the parameter

Reply:
I guess it is done.
Please check if it is done in the way you suggested.